metricbeat/module/kibana/stats/_meta/test/settings.700.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
The settings API in Kibana is not expected to exist until 6.5. Once it has been implemented for 6.5, I can add a `settings.650.json` file here. Should I do that?

Is your concern about how Metricbeat >= 6.5.0 will react when running with Kibana < 6.5.0?

Reply:
Yes. Seems like we should get the version number of KB on the first request and make decisions based on it.

Reply:
Actually I have a question about this. Prior to 6.4.0 there is no `GET /api/stats` in Kibana either. Does that mean we should be checking the version of Kibana before making _that_ call too?

Reply:
Yes, I think we will need such a check or at least provide a good error message.

I think we could check the version once during `New` and just assume it stays the same over the lifetime of the metricset for now. So we would only do it once.

Reply:
I'm going to add this version check in `New` in a separate PR since it affects code that has already been merged into `master` (#7525). I will rebase this PR on top of my new PR.

Reply:
Version check PR: https://github.com/elastic/beats/pull/7697

Reply:
Implemented version check for Kibana settings API in 2eaa84a.